.semaphore/semaphore.yml

```
@@ -0,0 +1,39 @@
+version: v1.0
+name: build-test-release
+agent:
+  machine:
+    # 4 CPU cores, 8 G RAM, 80G disk (Ubuntu 20.04)
+    type: s1-prod-ubuntu20-04-amd64-1
```

If you don't care about arch, could you use ARM? It's up to 20% faster and 20% cheaper.
```suggestion
    type: s1-prod-ubuntu20-04-arm64-1
```

---

Thanks; Github wouldn't let me apply your suggestion.. did it manually.